frontend/src/components/PreviewCard.js

```diff
@@ -9,15 +9,15 @@ const PreviewCard = (props) => {
         <img className="css-shadow" src={props.userPhotoUrl} alt="Image Not Avaliable" />
         <h5>By: {props.username}</h5>
       </div>
-      <a href=" ">
+      <span >
         <Resource type={props.type} link={props.src} className=" w-full" />
-      </a>
+      </span>

```

Comment:
change <a> to span because of console errors, cannot put <a> inside <a>. This does not affect functionality